arb/load_file.c

```diff
@@ -0,0 +1,35 @@
+/*
+    Copyright (C) 2019 Julian Rüth
+
+    This file is part of Arb.
+
+    Arb is free software: you can redistribute it and/or modify it under
+    the terms of the GNU Lesser General Public License (LGPL) as published
+    by the Free Software Foundation; either version 2.1 of the License, or
+    (at your option) any later version.  See <http://www.gnu.org/licenses/>.
+*/
+
+#include <stdio.h>
+#include <string.h>
+
+#include "arb.h"
+
+int arb_load_file(arb_t x, FILE* stream)
```

Comment:
Same question here: This exists essentially twice more (for arf & mag). Where should a `char* str_load_file(FILE* stream)` go?